virtual_ecosystem/models/animal/functional_group.py

```diff
@@ -150,8 +155,10 @@ def import_functional_groups(
             f"Invalid header. Expected at least {expected_header}, but got {fg.columns}"
         )
 
-    functional_group_list = [
-        FunctionalGroup(
+    for row in fg.itertuples():
```

Comment:
Not a problem with this change, but you could have kept the previous version and just add the missing argument as:


```python
    functional_group_list = [
        FunctionalGroup(
            row.name,
            row.taxa,
            row.diet,
            row.metabolic_type,
            row.reproductive_environment,
            row.reproductive_type,
            row.development_type,
            row.development_status,
            row.offspring_functional_group,
            row.excretion_type,
            row.migration_type,
            row.vertical_occupancy,
            row.birth_mass,
            row.adult_mass,
            density=getattr(row, "density_individuals_m2", None),
            constants=constants,
        )
        for row in fg.itertuples()
    ]
```